tests/components/wled/test_number.py

```diff
@@ -47,7 +47,7 @@ async def test_speed_segment_change_state(
 ) -> None:
     """Test the value change of the WLED segments."""
     await hass.services.async_call(
-        NUMBER_DOMAIN,
+        Platform.NUMBER,
```

Comment:
I don't think we should use the platform enum in service calls since this api is not only for platforms.

Reply:
I'm assuming this applies to all service calls not just number?

Reply:
Yes, that's correct. :+1: